File: code/4a194.pas
pprogram Bai194;
  var a,b, n, x,k, dem : integer;
begin
  readln(a,b);
    k := 0;
  for n := a to b do 
      begin
      dem := 0;
      for x := 1 to n do if n mod x = 0 then dem := dem +1; 
      if dem = 3 then k := k +1; 
      end;
      writeln(k);
end.

File: code/4a194.cpp
#include <iostream>
using namespace std;

int main() 
{
    int a,b,n,dem,x,d;
    cin>>a>>b;
    d=0;
    for (n=a; n<=b; n++){
        dem=0;
        for (x=1; x<=n; x++)
            if (n%x==0)
                dem++;
        if (dem==3)
        d++;
    }
    cout<<d;
    return 0;
}